### PR TITLE
Hogwarts Lab 04/21

### PR DESCRIPTION
**Pull request by edawson1980**

### CODE REVIEW

**Comment by edawson1980** [comment]
comfortability: 3.9
completeness: 3

Overall I'm actually pretty comfortable with Rails.  I've just been having some difficulty keeping all of the steps straight (especially pluralization).  I feel confident with the concepts and ideas, however.

File: app/views/houses/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<h1><%= @house.name %></h1>
-<% @house.students.each do |student| %>
+<h1><%= @houses.name %></h1>
```
Comment (jsm13) [inline app/views/houses/show.html.erb:1]: `@house`

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
     <%= javascript_include_tag "application", "data-turbolinks-track" => true %>
     <%= csrf_meta_tags %>
-    <title>Hogwarts
+    <title>Hogwarts</title>
   </head>
   <body>
     <nav>
       <h1>Welcome to Hogwarts!</h1>
-      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", student_path %> | <%= link_to "Enroll", student_path %></h2>
+      <h2><%= link_to "View Houses", houses_path %> | <%= link_to "View Students", houses_students_path %> | <%= link_to "Enroll", houses_students_path %></h2>
```
Comment (jsm13) [inline app/views/layouts/application.html.erb:12]: want a top level route for students in `routes.rb` and to link to `students_path`

File: app/views/students/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <h1>Here are the students</h1>
 <% @students.each do |student| %>
   <div>
-    <%= link_to student, students_path(student) %> - <%= student.house %>
+    <%= link_to "Student", houses_students_path(@student) %> - <%= @student.house %>
```
Comment (jsm13) [inline app/views/students/index.html.erb:4]: Would want either students to be a top level resource and use `student_path(@student)` or as is `house_student_path(@student.house, @student)`

**Review by jsm13 — COMMENTED** [review]
Well done! Nesting the routes led to some funny paths but overall great job

**Pull request closed without merging**